slides-battle-react/package.json

```diff
@@ -11,12 +11,14 @@
     "react": "^16.14.0",
     "react-bootstrap": "^1.4.0",
     "react-dom": "^16.14.0",
+    "react-hook-form": "^6.14.1",
     "react-hot-loader": "^4.13.0",
     "react-redux": "^7.2.1",
     "react-redux-hooks": "^0.3.0",
     "react-router-config": "^5.1.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.4"
+    "react-scripts": "^3.4.4",
+    "styled-components": "^5.2.1"
```

Comment:
Styled-components is a perfect library to handle dynamic CSS and pre-styled component
You don't need .css file anymore and you don't have to handle specific class-name for your components, everythin' is done for you
Since I used this lib on previous projects, I am not able to get rid off this one 
https://styled-components.com/

Reply:
I would prefer to keep a css-in-file approach instead of a css-in-js in a first version of the application. We will see next for go to css-in-js